builtin/mainmenu/content/dlg_package.lua

```diff
@@ -205,11 +208,41 @@ local function get_formspec(data)
 
 	elseif current_tab == 2 then
 		local hypertext = info.info_hypertext.head .. info.info_hypertext.body
-
 		table.insert_all(formspec, {
 			"hypertext[0,0;", W, ",", tab_body_height - 0.375,
 			";info;", core.formspec_escape(hypertext), "]",
 		})
+	elseif current_tab == 3 then
+		if not package.reviews and not data.reviews_error and not data.reviews_loading then
+			data.reviews_loading = true
+
+			contentdb.get_package_reviews(package, function(reviews)
+				if not reviews then
+					data.reviews_error = true
+				end
+				ui.update()
+			end)
+		end
+
+		if package.reviews then
+			local hypertext = package.reviews.head .. package.reviews.body
+			hypertext = hypertext:gsub("<img name=\"?blank.png\"? ",
+					"<img name=\"" .. core.hypertext_escape(defaulttexturedir) .. "blank.png\" ")
+			hypertext = hypertext:gsub("<thumbsup>",
```

Comment:
Are these tags documented anywhere? An "API" reference would be very helpful to have here.

Reply:
This is now documented at https://content.luanti.org/help/api/

Reply:
Is this referenced anywhere? Even if so, would you please be so nice to include a note to the header of this file, such as:
```
-- Note: The API documentation can be found here: https://content.luanti.org/help/api/
```
?